browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1400,7 +1400,7 @@ Are you sure you want to do this?
         Browsing visibility
       </message>
       <message name="IDS_BRAVE_NEW_TAB_PRIVATE_VISIBILITY_DESCRIPTION" desc="Browsing visibility description shown when a user opens a private tab explaining private mode">
-        Note that -even in private windows- your browsing activity may still be visible to your ISP, network observers, or your employer (if you're using a work device/network).
+        Note that - even in private windows - your browsing activity may still be visible to your ISP, network observers, or your employer (if you're using a work device/network).
```

Comment:
I believe we should use double dash as well in that place `—` instead of `-`, right @timchilds?

Reply:
Official style is em dashes, no spaces. So:

`Note that—even in private windows—your browsing activity…`

Reply:
I've made the necessary adjustments to the text as requested. However, I would like to kindly mention that, although I understand the use of em dashes is grammatically correct in American English, as a non-native speaker, this style can be somewhat concerning. It has the potential to cause significant confusion for individuals whose primary language is not American English. Perhaps, a more universal and easily understood form could be achieved using commas instead.

Reply:
I agree with @simoarpe , we will definitely have confusions after translation to other languages.

Reply:
hey all, sorry I missed this comment. 

the localized side can be customized in the strings so unless something is hardcoded we should be able to edit by language

Reply:
At this point, wouldn't it be better to get rid of the dashes in favor of commas for all the languages? 
Benefits: 
- More consisted style across all the languages
- International users that keeps Brave in English (like myself) would find it easier to read. (In many languages dashes don't exist)